Distribution/Server/Features/Users.hs

```diff
@@ -468,6 +493,66 @@ userFeature  usersState adminsState
           errBadRequest "User deleted"
             [MText "Cannot disable account, it has already been deleted"]
 
+    serveUserManagement :: DynamicPath -> ServerPartE Response
+    serveUserManagement dpath =
+        do (UserName username) <- userNameInPath dpath
+           uid <- lookupUserName (UserName username)
+           guardAuthorised [IsUserId uid, InGroup adminGroup]
```

Comment:
Use `guardAuthorised_` here since you don't need the return value (which is the uid)